src/rshim.c

```diff
@@ -2274,9 +2276,9 @@ static int rshim_handle_ownership_transfer(rshim_backend_t *bd)
   bool has_req;
 
   if (bd->drop_mode) {
-    if (bd->force_cmd_pending) {
-      RSHIM_INFO("rshim%d executing Force command\n", bd->index);
-      bd->force_cmd_pending = 0;
+    if (rshim_force_cmd_pending[bd->index]) {
+      RSHIM_INFO("rshim%d executing Force (REQ Ownership) command\n", bd->index);
```

Comment:
The RSHIM_INFO will be logged. Maybe we could simplify the message a little bit? like
"rshim%d executing OwnershipReq command"?

Reply:
I changed Force to ReqOwnership yesterday since Req Ownership is more intuitive. But QA and DOCs have been using them a lot already. So I decided to keep "Force". I will remove the "REQ Ownership" part to avoid confusion.